<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayOptions</class>
 <widget class="QWidget" name="DisplayOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DisplayOptions</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Ok">
      <property name="text">
       <string>Ok</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>331</width>
     <height>444</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Render size</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,1,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="width">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4000</number>
        </property>
        <property name="value">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="height">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4000</number>
        </property>
        <property name="value">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>px</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Field of view</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSpinBox" name="angle">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>170</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Degrees</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Background</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="background">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.200000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Octrees</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Levels render</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="Levels">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Surface Type</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Undetermined</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Peak</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Planar</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Ridge</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Pit</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Saddle</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Valley</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string>Red</string>
       </property>
       <property name="icon">
        <iconset theme="QColorConstants::Svg::blue">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cyan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Magenta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Yellow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DarkRed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Coral</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CornflowerBlue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DarkOrange</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LavenderBlush</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LightBlue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Black</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QComboBox" name="comboBox_3">
      <item>
       <property name="text">
        <string>Red</string>
       </property>
       <property name="icon">
        <iconset theme="QColorConstants::Svg::blue">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cyan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Magenta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Yellow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DarkRed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Coral</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CornflowerBlue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DarkOrange</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LavenderBlush</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LightBlue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Black</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QComboBox" name="comboBox_4">
      <item>
       <property name="text">
        <string>Red</string>
       </property>
       <property name="icon">
        <iconset theme="QColorConstants::Svg::blue">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cyan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Magenta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Yellow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DarkRed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Coral</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CornflowerBlue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DarkOrange</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LavenderBlush</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LightBlue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Black</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QComboBox" name="comboBox_5">
      <item>
       <property name="text">
        <string>Red</string>
       </property>
       <property name="icon">
        <iconset theme="QColorConstants::Svg::blue">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cyan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Magenta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Yellow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DarkRed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Coral</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CornflowerBlue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DarkOrange</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LavenderBlush</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LightBlue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Black</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QComboBox" name="comboBox_6">
      <item>
       <property name="text">
        <string>Red</string>
       </property>
       <property name="icon">
        <iconset theme="QColorConstants::Svg::blue">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cyan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Magenta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Yellow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DarkRed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Coral</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CornflowerBlue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DarkOrange</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LavenderBlush</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LightBlue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Black</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QComboBox" name="comboBox_7">
      <item>
       <property name="text">
        <string>Red</string>
       </property>
       <property name="icon">
        <iconset theme="QColorConstants::Svg::blue">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cyan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Magenta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Yellow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DarkRed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Coral</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CornflowerBlue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DarkOrange</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LavenderBlush</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LightBlue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Black</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QComboBox" name="comboBox_1">
      <item>
       <property name="text">
        <string>Red</string>
       </property>
       <property name="icon">
        <iconset theme="QColorConstants::Svg::blue">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cyan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Magenta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Yellow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DarkRed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Coral</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CornflowerBlue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DarkOrange</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LavenderBlush</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LightBlue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Black</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Planar Threshold</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>1/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="planthresh">
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>/maxBound</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Light</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="follow">
      <property name="text">
       <string>Follow Camera (SpaceBar)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Ambient light</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QDoubleSpinBox" name="ambient">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>* Background</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
